### DISCUSSION TITLE
Why does editor-theme3/compatibility.css use different category names than scratch

### DISCUSSION THREAD

**Discussion by TheColaber** [ScratchAddons/ScratchAddons#5051] [Discussion]
cc @mxmou 

It seems like all the addons that use this need to create an object like this:
![image](https://user-images.githubusercontent.com/72760579/187294727-f227a5d0-56c6-460d-b317-a4e56565b790.png)
in order to match up the right class names. Why was this done?

**Comment by mxmou** [comment]
These names are the same as `Blockly.Colours` entries. `data_lists` could be renamed to `data-lists` and `event` to `events` to match the actual categories but some addons will still need a `colorIds` map.
